SubmittedAssignments/RohitBagade_Assignment77/css/style.css

```
@@ -0,0 +1,434 @@
+	border-left: 70mm solid #212b36;
+	border-bottom: 70mm solid transparent;
+	top: 0;
+	left: 0;
+	z-index: 2;
+}
+
+.top-left::after {
+	top: -50mm;
+	border-left: 80mm solid #e91a34;
+	border-bottom: 80mm solid transparent;
+	top: 0;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

top is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)